src/store/polls/actions.ts

```diff
@@ -32,4 +33,37 @@ export const actions: ActionTree<PollsStateInterface, RootStateInterface> & Acti
       console.log(e.response);
     }
   },
+  [PollsActionEnum.DELETE_POLL]: async ({ commit }, payload) => {
+    try {
+      const url = `cooperations/${payload.cooperationId}/polls/${payload.pollId}`;
+      await HTTP.delete(url);
+      commit(PollsMutationEnum.DELETE_POLL, payload.pollId);
+    } catch (e: any) {
+      console.log(e.response);
+    }
+  },
+  [PollsActionEnum.UPDATE_POLL]: async ({ commit }, payload) => {
+    try {
+      const pollToSend = new PostPollDTOModel(payload.data);
+      const url = `cooperations/${payload.ids.cooperationId}/polls/${payload.ids.pollId}`;
+      const { data } = await HTTP.put(url, pollToSend);
+
+      // const mockData = {
+      //   id: data.id,
+      //   header: data.header,
+      //   description: payload.data.description,
+      //   creation_date: payload.data.creationDate,
+      //   completion_date: payload.data.completionDate,
+      //   status: data.status,
+      //   type: data.type,
+      //   polled_houses: payload.data.polledHouses,
+      // };
```

Comment:
Do you still need this commented code?

Reply:
Actually yes. This PR is draft, so I commented the part of code for a while